drakrun/drakrun/draksetup.py

```diff
@@ -8,6 +9,7 @@
 import json
 import time
 import random
+import secrets
```

Comment:
It seems to be better cryptographically instead of random, so changed it for password update

Reply:
Also, the initial password is used from the template file only, should that also be changed?

Reply:
I think it's ok.